projects/storefrontlib/shared/components/list-navigation/pagination/pagination.component.ts

```diff
@@ -37,9 +37,11 @@ export class PaginationComponent {
   get pagination(): PaginationModel {
     return this._pagination;
   }
-  @Input() set pagination(value: PaginationModel) {
-    this._pagination = value;
-    this.render(value);
+  @Input() set pagination(value: PaginationModel | undefined) {
```

Comment:
Is there a way to not accept `undefined` if we are not goint to do anything when we recieve it?

Reply:
`pagination` defined in interfaces are `optional`. The internal one is this `_pagination: PaginationModel`. So, if `undefined` is passed, it just not do rendering